saltyrtc/server/events.py

```diff
@@ -0,0 +1,25 @@
+import collections
+from collections.abc import Coroutine
+from enum import (
+    Enum,
+    unique,
+)
+from typing import List
+
+__all__ = ('Event',)
+
+
+@unique
+class Event(Enum):
+    INITIATOR_CONNECTED = 'initiator-connected'
+    RESPONDER_CONNECTED = 'responder-connected'
```

Comment:
Either should be fine, both will be converted to hash values for lookup in the dict anyway, right?